content/spin/kv-store.md

```diff
@@ -339,6 +391,55 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 {{ blockEnd }}
 
+{{ startTab "Python"}}
+
+<!-- @nocpy -->
+
+```typescript
+import { HandleRequest, HttpRequest, HttpResponse } from "@fermyon/spin-sdk"
+
+const encoder = new TextEncoder()
+const decoder = new TextDecoder()
+
+export const handleRequest: HandleRequest = async function (request: HttpRequest): Promise<HttpResponse> {
+
+  let store = spinSdk.kv.openDefault()
+  let status = 200
+  let body
+
+  switch (request.method) {
+    case "POST":
+      store.set(request.uri, request.body || (new Uint8Array()).buffer)
+      break;
+    case "GET":
+      let val
+      try {
+        val = store.get(request.uri)
+        body = decoder.decode(val)
```

Comment:
Does this need decoding?  It looks like POST stuffs the body bytes into the store - it could be an image or generic blob, not necessarily a string I think?